File: sephora-frontend/src/models/category/CategoryDto.ts
import PictureDto from "../picture/PictureDto.ts";

export default interface CategoryDto {
    get id(): number;

    get nameEn(): string;

    get nameUa(): string;

    get descriptionEn(): string;

    get descriptionUa(): string;

    get picture(): PictureDto;
}

File: sephora-backend/Application/Dtos/Category/CategoryDTO.cs
namespace CleanArchitecture.Application.Dtos.Category;

public class CategoryDto
{
    public int Id { get; set; }
    
    public string NameEn { get; set; } = String.Empty;
    
    public string NameUa { get; set; } = String.Empty;
    
    public string DescriptionEn { get; set; } = String.Empty;
    
    public string DescriptionUa { get; set; } = String.Empty;

    public PictureDto Picture { get; set; } = default!;
}
